src/main/java/org/terasology/smithing/component/CharcoalPitComponent.java

```diff
@@ -19,16 +19,28 @@
 import org.terasology.network.Replicate;
 
 public class CharcoalPitComponent implements Component {
+
+    /** The time in milliseconds in-game when the charcoal pit begins to burn **/
     @Replicate
     public long burnStartWorldTime;
+
+    /** The time in milliseconds in-game when the charcoal pit stops burning **/
```

Comment:
Same note as the begin time. Until the burning stops what is a likely value here? `-1` ? Or the last time or something?